terraform/terraform.tfvars

```diff
@@ -29,8 +29,14 @@ clusters = {
       kubernetes_version = "1.25"
       release_channel    = "UNSPECIFIED" # "REGULAR"
 
-      enable_monitoring  = true
-      enable_logging     = true
+      enable_monitoring = true
+      enable_logging    = true
+
+      # If you want to specify a specific service account, set
+      # create_service_account to false, and specify the service_account.
+      create_service_account = true
+      service_account_name   = ""
+      service_account        = ""
```

Comment:
Added these lines.

Reply:
These settings should also be added to the other GKE configs in terraform.tfvars.  Also, these options need to be added to modules/gke/variables.tf and variables.tf.  I'm not sure how this could have worked otherwise.  Terraform should have complained about invalid options during apply otherwise.

Reply:
Thanks. That was dumb of me. I'll make a new PR.

Reply:
Lastly, why are we adding this?  Are we certain we want to do this?  I believe the same thing can be accomplished via environment variables.  I already feel like we are bordering on having too much in the config options and I'd hate to add something that might be confusing to novice users.